brave/src/test/java/com/linecorp/armeria/client/brave/BraveClientIntegrationTest.java

```diff
@@ -163,7 +164,7 @@ protected void getAsync(HttpClient client, String pathIncludingQuery) throws Exc
     private static class DummyRequestContext extends NonWrappingRequestContext {
         DummyRequestContext() {
             super(NoopMeterRegistry.get(), SessionProtocol.HTTP,
-                  HttpMethod.GET, "/", null, HttpRequest.streaming(HttpMethod.GET, "/"), null);
+                  UUID.randomUUID(), HttpMethod.GET, "/",  null, HttpRequest.streaming(HttpMethod.GET, "/"), null);
```

Comment:
Could you remove extra space after `"/",`?

Reply:
This is not fixed yet.